awxkit/awxkit/api/pages/inventory.py

```diff
@@ -32,7 +32,7 @@
 class Inventory(HasCopy, HasCreate, HasInstanceGroups, HasVariables, base.Base):
 
     dependencies = [Organization]
-    NATURAL_KEY = ('organization', 'name')
+    NATURAL_KEY = ('organization', 'hosts', 'groups', 'name')
```

Comment:
Hosts and groups should not be part of the natural key here.  The natural key is supposed to be a (mostly) fixed minimal subset of the information of the object needed to uniquely look it up.